sdk/python/lib/pulumi/runtime/mocks.py

```diff
@@ -167,8 +173,7 @@ def ReadResource(self, request):
                                          name=request.name,
                                          inputs=state,
                                          provider=request.provider,
-                                         resource_id=request.id,
-                                         custom=request.custom or False)
```

Comment:
request.custom is not propagated here - that is on purpose? Thanks.

Reply:
Yeah, that's the fix actually. The [ReadResource](https://github.com/pulumi/pulumi/blob/master/sdk/python/lib/pulumi/runtime/resource.py#L347-L359) request does not contain a `custom` attribute.